ttps/discovery/macos/get-running-processes/README.md

```diff
@@ -0,0 +1,23 @@
+# macOS Get Running Processes
+
+![Meta TTP](https://img.shields.io/badge/Meta_TTP-blue)
+
+Utilize ps to view currently running processes.
+
+## Pre-requisites
+
+Ensure that ps is installed on the target system and that you have the necessary permissions to run it.
+
+## Examples
+
+Utilize ps to view currently running processes. This TTP does not produce artifacts,
+so it is not necessary to run the cleap up afterward:
+
+```bash
+ttpforge -c config.yaml \
```

Comment:
hey! see if it works without the `-c config.yaml` - added some code to smooth out the CLI a bit, should work with just the path